lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -41,6 +48,12 @@ extension LocalizedBody on Event {
             ?.toUpperCase());
   }
 
+  bool get isVideoOrImage =>
```

Comment:
why just 
```
messageType == MessageTypes.Image && messageType == MessageTypes.Video
```

Reply:
same result, array is better if you want to add more case in the future